### DISCUSSION TITLE
Can I develop an Android application not the webapp (as shown in demos) which uses the AI search and Azure Open AI rest apis? Is that applicable for hack app?

### DISCUSSION THREAD

**Discussion by vimisms** [microsoft/AI-Chat-App-Hack#58] [Q&A] [answered]
I am new to AI and have limited knowledge of python. Wanted to check if android application which uses AI search and open AI rest apis applicable for this hackathon.

**Comment by vimisms** [comment]
Also wanted to check if I can manually perform the chunking and generate vectors (not with the code due to limited knowledge) will be considered.

**Reply by pamelafox** [reply, marked as answer]
If you use Azure in some way and AI in some way, that fits the theme. Looking forward to seeing it!
